### PR TITLE
[bundle-size] Remove legacy code and replace with per-file approval logic

### PR DESCRIPTION
**Pull request by danielrozenberg**
#617

### CODE REVIEW

File: bundle-size/api.js [hunk 1]
```diff
@@ -98,20 +80,20 @@ function successfulCheckOutput(
  * @return {!Octokit.ChecksCreateParamsOutput} check output.
  */
 function erroredCheckOutput(partialBaseSha) {
+  const superUserTeams = `@${process.env.SUPER_USER_TEAMS.replace(/,/, ', @')}`;
```
Comment (rcebulko) [inline bundle-size/api.js]: `/,/g`

File: bundle-size/webhooks.js [hunk 2]
```diff
@@ -108,48 +110,31 @@ exports.installGitHubWebhooks = (app, db, githubUtils) => {
...
-        approverTeams
-      );
-    }
-
-    // TODO(#617, danielrozenberg): remove the legacy logic below.
-    if (!(await githubUtils.isBundleSizeApproverLegacy(approver))) {
-      return;
-    }
-
+    const isApprover = (
```
Comment (rcebulko) [inline bundle-size/webhooks.js]: `isSuperApprover || (...)` to potentially save API calls and such

File: bundle-size/api.js (hunk 1 above)
Comment (danielrozenberg) [reply bundle-size/api.js]: Done

File: bundle-size/webhooks.js (hunk 2 above)
Comment (danielrozenberg) [reply bundle-size/webhooks.js]: Done

File: bundle-size/test/api.test.js [hunk 3]
```diff
@@ -418,153 +446,36 @@ describe('bundle-size api', () => {
...
-                resolve(nocks);
-              }, 0);
-            });
-        });
+            }
+          )
+          .reply(200, () => {
+            setTimeout(() => {
+              resolve(nocks);
+            }, 0);
```
Comment (rcebulko) [inline bundle-size/test/api.test.js]: `setTimeout(cb)` defaults to a 0 timeout; the 0 isn't required.

Comment (danielrozenberg) [reply bundle-size/test/api.test.js]: TIL!

**Pull request merged**